test/rambla_test.exs

```diff
@@ -133,7 +133,7 @@ defmodule Test.Rambla do
   end
 
   test "modern works with reliable rabbit" do
-    Rambla.Handlers.Amqp.publish(:reliable_channel, %{message: %{foo: 42}, exchange: "rambla-exchange-1"})
+    Rambla.publish(:reliable_channel, %{message: %{foo: 42}, exchange: "rambla-exchange-1"})
```

Comment:
The main advantage of `Rambla` (besides the transparent distributed retry mechanism) is it is agnostic to actual implementations.

You can attach as many handlers to the channel with that particular name and then `Rambla.publish/2` will deliver it to _all_ the configured destinations.